rtsp_to_hls/hls_proxy_player/player.html.template

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <script src="https://cdn.jsdelivr.net/npm/hls.js@latest"></script>
+    <title>HLS Proxy player</title>
+  </head>
+
+  <body style="margin:0">
+      <main role="main">
+        <video controls id="video" style="width:100%;height:auto" muted="muted"></video>
+        <script>
+          console.log(Hls.isSupported());
```

Comment:
I would add more informational message, something like: `Is HLS supported: <true|false>`

Reply:
done